nosqlbench/http-jsonapi-crud-basic.yaml

```diff
@@ -49,6 +49,7 @@ blocks:
         X-Cassandra-Request-Id: "{request_id}"
         X-Cassandra-Token: "<<auth_token:my_auth_token>>"
         Content-Type: "application/json"
+        ok-body: ".*\"ok\":1.*"
```

Comment:
these `ok-body` changes make the tests a bit more robust - since the JSON API returns 200 for most cases, we need to make sure the output has expected keys.